serfclient/client.py

```diff
@@ -74,3 +74,11 @@ def stats(self):
         Obtain operator debugging information about the running Serf agent.
         """
         return self.connection.call('stats')
+
+    def shutdown(self):
```

Comment:
Rather than `shutdown` a more consistent method name here is probably `close`. Once you've made this change, I'm happy to merge this in 👍 . Thanks for the PR! 👏